ci/doc.yml

```diff
@@ -13,11 +13,10 @@ dependencies:
   - properscoring
   - matplotlib-base
   - scipy
-  - sphinx
+  - sphinx<4.4
```

Comment:
why's this needed?

Reply:
See the commit before which failed but I don't know why...

Reply:
The last commmit in main was before 4.4

Reply:
made a note to take a look another time https://github.com/xarray-contrib/xskillscore/issues/372